### PR TITLE
Add WebGL setup guide

### PR DESCRIPTION
**Pull request by lexaknyazev**

### CODE REVIEW

File: KTXDeveloperGuideWebGL.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Modern Intel GPUs, namely Intel HD Graphics 5xx ("Skylake") or newer, have the best compressed format coverage on Windows and Linux. OpenGL or Vulkan are required to get access to ASTC and ETC formats.
+
+1. Consider using the latest GPU driver. On Windows, it could be downloaded from the [Intel website](https://downloadcenter.intel.com/); on Linux, it is usually provided and maintained by OS distribution vendors.
+
+2. Install [Google Chrome Dev](https://www.google.com/chrome/dev/) or [Microsoft Edge Dev](https://www.microsoftedgeinsider.com/en-us/download/).
+
+3. On Windows, open `chrome://flags/#use-angle` page, select OpenGL as ANGLE graphics backend, and restart the browser.
+
+4. Confirm enabled WebGL extensions here: https://webglreport.com/.
```
Comment (donmccurdy) [inline KTXDeveloperGuideWebGL.md:17]: Can we share an example of a WebGL extension users should expect to see, if everything worked so far? `WEBGL_compressed_texture_astc`?

File: KTXDeveloperGuideWebGL.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## macOS
+
+Apple M1 SoC supports all possible compressed transcode targets: BC*, ETC*, ASTC, and PVRTC1. Most of them are available only via Metal API. Intel-based Macs support only BC* formats.
+
+1. Install [Google Chrome Dev](https://www.google.com/chrome/dev/) or [Microsoft Edge Dev](https://www.microsoftedgeinsider.com/en-us/download/).
+
+2. Run it with the following cmd-line parameters: `--use-cmd-decoder=passthrough --use-gl=angle --use-angle=metal`.
+
+3. Confirm enabled WebGL extensions here: https://webglreport.com/.
```
Comment (donmccurdy) [inline KTXDeveloperGuideWebGL.md:27]: See above.

File: KTXDeveloperGuideWebGL.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Platform Support Table
```
Comment (donmccurdy) [inline KTXDeveloperGuideWebGL.md:29]: Might be helpful to include the relevant WebGL extensions in the table, if they'll fit. Or just link to https://github.com/KhronosGroup/3D-Formats-Guidelines/blob/main/KTXDeveloperGuide.md#gpu-api-support?

**Review by donmccurdy — APPROVED** [review]
A few minor suggestions, but feel free to merge this with or without changes. Thanks!

File: KTXDeveloperGuideWebGL.md (hunk 3 above)
Comment (lexaknyazev) [reply KTXDeveloperGuideWebGL.md:29]: I'll add the extension names to the steps above.

**Review by donmccurdy — APPROVED** [review]

**Pull request merged**